stablehlo/dialect/TypeInference.cpp

```diff
@@ -666,10 +674,11 @@ LogicalResult verifyReduceWindowOpInputsAndInferWindow(
             " is not compatible with shape at input-index ", rankedInputIdx);
   }
 
-  // P2.
+  // reduce_window_i3
   auto windowDimsOrErr =
       convert1DAttribute(windowDimensions, location, "window_dimensions");
   if (failed(windowDimsOrErr)) return failure();
+  // reduce_window_c4
```

Comment:
What do you think about moving the reduce_window_c4 code block after all reduce_window_ix code blocks, so that we do i3..i7, then c4, and then c5..c12 (via verifyWindowAttributesAndInferWindowDimensions)?